uqcsbot/scripts/wolfram.py

```diff
@@ -0,0 +1,192 @@
+from uqcsbot import bot, Command
+from typing import Iterable, Tuple
+import requests
+import json
+
+APP_ID = "UVKV2V-5XW2TETT69"
+
+# IDEA: Show the assumptions made?
+# TODO: Better naming of commands
+@bot.on_command("wolframfull")
+async def handle_wolframfull(command: Command):
+    """
+    This posts the full results from wolfram query. Images and all
+
+    Example usage:
+    !wolframfull y = 2x + c
+    """
+    api_url = r"http://api.wolframalpha.com/v2/query?&output=json"
+    search_query = command.arg
+    http_response = await bot.run_async(requests.get, api_url, params={'input': search_query, 'appid': APP_ID})
+
+    # Check if the response is ok
+    if http_response.status_code != requests.codes.ok:
+        bot.post_message(command.channel, "There was a problem getting the response")
+        return
+
+    # Get the result of the query and determine if wolfram succeeded in evaluating it
+    result = json.loads(http_response.content)['queryresult']
+    if not result['success'] or result["error"]:
+        bot.post_message(command.channel, "Please rephrase your query. Wolfram could not compute.")
+
+    subpods = get_subpods(result['pods'])
```

Comment:
this needs renaming/a descriptive comment but I'll reference it in the function comments :P

Reply:
If you don't rename it, add comments here to be explicit I guess